src/styles/blocks/card.scss

```
@@ -0,0 +1,101 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid $grey-elements;
+  border-color: $white-elements;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  word-break: break-word;
+  
+  
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
```

you'd better reset margin for `h2` and `p` the same as you did with `body` and use only `margin-bottom` here
```suggestion
    margin-bottom: 4px;
```